operator/exporter.py

```diff
@@ -0,0 +1,180 @@
+import json
+import urllib3
+import uvicorn
+import logging
+from utils import execute, CommandError
+from fastapi import FastAPI
+from prometheus_client import make_asgi_app, start_http_server, Gauge
+from kadalulib import logf, logging_setup
+
+# Define all metrics here
```

Comment:
Move all these definitions to `metrics.py` and import here. and use as `metrics.total_capacity_bytes`...